.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "src/components/lib/ck/ck"]
 	path = src/components/lib/ck/ck
 	url = https://github.com/gwsystems/ck.git
+[submodule "src/components/lib/sqlite/sqlite"]
```

Comment:
You'll want to move this into gwsystems. Remember, we want the main branch of the repo be unmodifed, and have a `cos` branch that we pull in as a submodule. That way we have a path for updating the code as upstream changes.